gramex/apps/guide/gramextutorials/1_building_interactive_dashboards.md

```diff
@@ -0,0 +1,228 @@
+---
```

Comment:
Feedback from @mankoven :

1. Add interactive elements in the charts (see: https://uat.gramener.com/gramex-workshop-01)

Reply:
2. Add more charts from the repo for above ^